src/components/Header.js

```diff
@@ -1,14 +1,134 @@
 import React from 'react';
 import styled from 'styled-components';
 
+import bblfshLogo from '../img/bblfsh.png';
+import githubIcon from '../img/github.svg';
+
 const Container = styled.header`
   height: 70px;
+  padding: 0 1rem;
+  display: flex;
+  align-items: center;
+  border-bottom: 1px solid black;
+`
+
+const Title = styled.h1`
+  display: flex;
+  align-items: center;
+`
+
+const TitleImage = styled.img`
+  height: 70px;
+`
+
+const DashboardTitle = styled.span`
+  font-weight: bold;
+  font-size: 1.5rem;
+  margin-left: .5rem;
 `
 
-export default function Header() {
+const Actions = styled.div`
+  width: 100%;
+  display: flex;
+  margin-left: 1rem;
+`
+
+const InputGroup = styled.div`
+  display: flex;
+  align-items: center;
+
+  & + & {
+    margin-left: 1rem;
+  }
+
+  & > label {
+    margin-right: .5rem;
+  }
+`
+
+const RunButton = styled.button`
+  padding: .2rem 1.5rem;
+  border-radius: 3px;
+  border: none;
+  cursor: pointer;
+`
+
+const DriverCodeBox = styled.div`
+  display: flex;
+  align-items: center;
+  margin-left: .5rem;
+`
+
+const DriverCodeText = styled.a`
+  margin-left: .5rem;
+`
+
+export function DriverCode({ languages, selectedLanguage, actualLanguage }) {
+  const driver = selectedLanguage === 'auto' 
+    ? actualLanguage
+    : selectedLanguage;
+
+  return (
+    <DriverCodeBox>
+      <img src={githubIcon} alt='driver code' />
```

Comment:
the `alt` should be something describing the image itself, or empty if its not important for the content itself
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img